src/test/java/me/itzg/testing/DockerRuleTestElasticsearch.java

```diff
@@ -0,0 +1,83 @@
:0,\"total_count\":0,\"hit_count\":0,\"miss_count\":0,\"cache_size\":0,\"cache_count\":0,\"evictions\":0},\"fielddata\":{\"memory_size_in_bytes\":0,\"evictions\":0},\"completion\":{\"size_in_bytes\":0},\"segments\":{\"count\":0,\"memory_in_bytes\":0,\"terms_memory_in_bytes\":0,\"stored_fields_memory_in_bytes\":0,\"term_vectors_memory_in_bytes\":0,\"norms_memory_in_bytes\":0,\"points_memory_in_bytes\":0,\"doc_values_memory_in_bytes\":0,\"index_writer_memory_in_bytes\":0,\"version_map_memory_in_bytes\":0,\"fixed_bit_set_memory_in_bytes\":0,\"max_unsafe_auto_id_timestamp\":-1,\"file_sizes\":{}},\"translog\":{\"operations\":0,\"size_in_bytes\":215},\"request_cache\":{\"memory_size_in_bytes\":0,\"evictions\":0,\"hit_count\":0,\"miss_count\":0},\"recovery\":{\"current_as_source\":0,\"current_as_target\":0,\"throttle_time_in_millis\":0}},\"total\":{\"docs\":{\"count\":0,\"deleted\":0},\"store\":{\"size_in_bytes\":324,\"throttle_time_in_millis\":0},\"indexing\":{\"index_total\":0,\"index_time_in_millis\":0,\"index_current\":0,\"index_failed\":0,\"delete_total\":0,\"delete_time_in_millis\":0,\"delete_current\":0,\"noop_update_total\":0,\"is_throttled\":false,\"throttle_time_in_millis\":0},\"get\":{\"total\":0,\"time_in_millis\":0,\"exists_total\":0,\"exists_time_in_millis\":0,\"missing_total\":0,\"missing_time_in_millis\":0,\"current\":0},\"search\":{\"open_contexts\":0,\"query_total\":0,\"query_time_in_millis\":0,\"query_current\":0,\"fetch_total\":0,\"fetch_time_in_millis\":0,\"fetch_current\":0,\"scroll_total\":0,\"scroll_time_in_millis\":0,\"scroll_current\":0,\"suggest_total\":0,\"suggest_time_in_millis\":0,\"suggest_current\":0},\"merges\":{\"current\":0,\"current_docs\":0,\"current_size_in_bytes\":0,\"total\":0,\"total_time_in_millis\":0,\"total_docs\":0,\"total_size_in_bytes\":0,\"total_stopped_time_in_millis\":0,\"total_throttled_time_in_millis\":0,\"total_auto_throttle_in_bytes\":104857600},\"refresh\":{\"total\":0,\"total_time_in_millis\":0,\"listeners\":0},\"flush\":{\"total\":0,\"total_time_in_millis\":0},\"warmer\":{\"current\":0,\"total\":5,\"total_time_in_millis\":0},\"query_cache\":{\"memory_size_in_bytes\":0,\"total_count\":0,\"hit_count\":0,\"miss_count\":0,\"cache_size\":0,\"cache_count\":0,\"evictions\":0},\"fielddata\":{\"memory_size_in_bytes\":0,\"evictions\":0},\"completion\":{\"size_in_bytes\":0},\"segments\":{\"count\":0,\"memory_in_bytes\":0,\"terms_memory_in_bytes\":0,\"stored_fields_memory_in_bytes\":0,\"term_vectors_memory_in_bytes\":0,\"norms_memory_in_bytes\":0,\"points_memory_in_bytes\":0,\"doc_values_memory_in_bytes\":0,\"index_writer_memory_in_bytes\":0,\"version_map_memory_in_bytes\":0,\"fixed_bit_set_memory_in_bytes\":0,\"max_unsafe_auto_id_timestamp\":-1,\"file_sizes\":{}},\"translog\":{\"operations\":0,\"size_in_bytes\":215},\"request_cache\":{\"memory_size_in_bytes\":0,\"evictions\":0,\"hit_count\":0,\"miss_count\":0},\"recovery\":{\"current_as_source\":0,\"current_as_target\":0,\"throttle_time_in_millis\":0}}}}}");
```

Comment:
This seems like a brittle assertion...and in fact it failed when I ran it. I never could narrow down which specific bit differed. I would suggest you instead do some spot assertions of fields within the `indicesStatsResponse` object itself.